src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -31,6 +33,26 @@ type WorkspaceTagsSettingsPageOnyxProps = {
 };
 type WorkspaceTagsSettingsPageProps = WorkspaceTagsSettingsPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAGS_SETTINGS>;
 
+/**
+ * pending state of this component can be achieved using setPolicyBillableMode or disableWorkspaceBillableExpenses functions;
+ * setPolicyBillableMode changes both disabledFields and defaultBillable and is called when disabledFields.defaultBillable is set
+ * otherwise disableWorkspaceBillableExpenses is used and it changes only disabledFields
```

Comment:
```suggestion
 * The pending state might be set by either setPolicyBillableMode or disableWorkspaceBillableExpenses.
 * setPolicyBillableMode changes disabledFields and defaultBillable and is called when disabledFields.defaultBillable is set.
 * Otherwise, disableWorkspaceBillableExpenses is used and it changes only disabledFields
```

Reply:
Great, I think it looks better now 🌟